### DISCUSSION TITLE
Installed on my new computer but Windows 11 start menu is still there

### DISCUSSION THREAD

**Discussion by 24JGM** [Open-Shell/Open-Shell-Menu#1556] [General]
Hi all, sincerest apologies if this is in the wrong place.

I've installed Open-Shell on many previous computers with little issue, but this one is throwing me for a loop. I downloaded version 4.4.170, and it installed no problem, but it's not actually doing anything. The factory task bar remains unfazed even though Open-Shell Menu is running in Task Manager (see attached screenshots). This is the first Windows 11 computer I've dealt with, so I thought that the OS might be at the root of the issue, but looking online, I see that others with 11 aren't having this issue. I've tried uninstalling and reinstalling to no avail, so I'm really flummoxed with this to put it mildly.

My thanks to any who can help me be rid of this horrid Windows 11 Start Menu.

Windows 11 Home, version 22H2, OS build 22621.1778

<img width="640" alt="Untitled" src="https://github.com/Open-Shell/Open-Shell-Menu/assets/135292356/aeaea477-51b7-439f-9a38-3504ad07768f">
<img width="640" alt="Untitledw" src="https://github.com/Open-Shell/Open-Shell-Menu/assets/135292356/9ff7c42c-ef02-4a65-a113-ff981406baf2">

**Comment by tallman69** [comment]
Uninstall 4.4.170 and install 4.4.189 instead.  4.4.189 is a prerelease that works better on Windows 11 than previous releases of OpenShell.  I am currently using 4.4.189 on my two windows 11 machines without issue.

**Comment by 24JGM** [comment]
SOLVED
Thanks!
